### PR TITLE
Add Filebeat Retries

### PR DESCRIPTION
**Pull request by ddaskal**

### CODE REVIEW

File: roles/logging/tasks/filebeat.yml [hunk 1]
```diff
@@ -4,10 +4,14 @@
     pkg: "filebeat={{ logging.forward.version }}"
   notify: restart filebeat
   when: os == "ubuntu"
-  
+
 - name: install filebeat yum
   yum:
     name: "filebeat"
+  register: result
```
Comment (paulczar) [inline roles/logging/tasks/filebeat.yml]: for shits and giggles add the retry for ubuntu above as well.   I'm told ubuntu repos are also a trash fire.

File: roles/logging/tasks/filebeat.yml [hunk 2]
```diff
@@ -52,3 +56,7 @@
 
 - name: start and enable filebeat service
   service: name=filebeat state=started enabled=yes
+  register: result
```
Comment (paulczar) [inline roles/logging/tasks/filebeat.yml]: probably don't need a retry on the service start?  unless you've seen issues here ?

File: roles/logging/tasks/filebeat.yml (hunk 1 above)
Comment (ddaskal) [inline roles/logging/tasks/filebeat.yml]: done

File: roles/logging/tasks/filebeat.yml (hunk 2 above)
Comment (ddaskal) [inline roles/logging/tasks/filebeat.yml]: I hit issues here today, adding retry made it succeed on the 3rd run. Will investigate logs and find more elegant solution today

**Comment by ddaskal** [comment]
retest

**Pull request merged**